apps/web/components/signin/CredentialsForm.tsx

```diff
@@ -208,7 +207,6 @@ function SignUp() {
           <ActionButton
             type="submit"
             loading={form.formState.isSubmitting}
-            disabled={clientConfig.auth.disableSignups}
           >
             Sign Up
```

Comment:
How about we just do the change in the text directly here?:
```suggestion
            {clientConfig.auth.disableSignups ? "Signups are currently disabled" : "Sign Up"}
```

Reply:
Hiding the whole signup form or tab would be much clearer, I don't think we need the form shown if it serve no purposes other than confusing users.

Reply:
I agree, this also confused me when i had that set for testing purposes. If signup is disabled, I don't expect to see a signup page

Reply:
Ok, in that case I don't mind.